imx.mk

```diff
@@ -11,6 +11,7 @@ override COMPILE_S_KERNEL  := 64
 OPTEE_OS_PLATFORM = imx-mx8mqevk
 BR2_TARGET_GENERIC_GETTY_PORT = ttymxc0
 BR2_TARGET_ROOTFS_EXT2=y
+BR2_TARGET_ROOTFS_EXT2_4=y
```

Comment:
Does this mean we're generating both rootfs.ext2 and rootfs.ext4?

Reply:
Hey! The build process creates these two files, but `rootfs.ext4` is a symlink to `rootfs.ext2`. Executing `file` on `rootfs.ext2` returns `Linux rev 1.0 ext4 filesystem data`. Therefore, only the ext3 filesystem image is created.

The flag `BR2_TARGET_ROOTFS_EXT2` is required to enable the building of an ext2/3/4 root filesystem.

Reply:
OK